lib/Interpreter/CppInterOp.cpp

```diff
@@ -1223,6 +1223,24 @@ namespace Cpp {
     GetClassDecls<VarDecl>(scope, datamembers);
   }
 
+  void GetEnumConstantDatamembers(TCppScope_t scope,
+                                  std::vector<TCppScope_t>& datamembers,
+                                  bool include_class_enum) {
+    std::vector<TCppScope_t> EDs;
+    GetClassDecls<EnumDecl>(scope, EDs);
+    for (TCppScope_t i : EDs) {
+      auto* ED = static_cast<EnumDecl*>(i);
+
+      bool is_class_tagged = ED->isScopedUsingClassTag();
+      if ((is_class_tagged) && !(is_class_tagged && include_class_enum))
```

Comment:
```suggestion
      if (is_class_tagged || !include_class_enum)
```
Isn't that what you mean? Only include scoped enums when the user requested?